### ISSUE TITLE
LiveShare Questions for VS Code Accessibility Call-July2020

### ISSUE THREAD

**Issue by venkateshpotluri** [microsoft/vscode#103116]
Hi everybody,

We would like to know more about your experience with LiveShare and your experience remotely collaborating on code in the July VSCode accessibility call. You are not required to use LiveShare; we would still like to hear from you! If you choose to try LiveShare, our conversations may be centered around the follow feature of LiveShare. Simply put, the follow feature lets a collaborator (a follower) follow the host (leader)'s cursor. We look forward to talking to you! Hear are some concepts, and a tentative list of questions.

## Terminology
* A leader is a person who may be an expert of a particular piece of code. In live share, this may be a person who is hosting a live share session.
* A follower is a person who is trying to walk through the code with the leader, with the objective of learning how a system works, understanding a code change, etc.
* A code walkthrough is a guided exploration of source code, where one person is the leader who explains the code to a follower. It may be part of a formal process, such as a code review, or just an informal discussion. Today, we will talk about code walkthroughs, using Live Share in VSCode.

## Questions

1.	How do you participate in live, collaborative programming activities like code walkthroughs? 
  1. How often do you participate in code walkthroughs?
  2.	Do your collaborators include blind, low vision, and sighted people? What percentage of each in all?
  
2. Have you tried LiveShare in VSCode?
    1. What were you trying to accomplish when using Live Share?
        1.	Were you able to accomplish this task successfully using live share features in VSCode?
        2.	Was this as efficient as you hoped?
        3.	What could make it better?
        4.	Was there missing information that you wish Live Share provided?
        5.	What was this missing information?
    2. Have you used the follow feature in live share?
        1. What was your experience like?
        2. If you are the follower, 
            * How was that experience for you?
            * If the other person is going too fast, how do you indicate this to them?
            * If the other person is going too slow, how do you indicate this to them?
            * What do you do when you are lost?
            * How would you communicate to the other person if you feel lost?
            * What have you done in the past to get unlost?
            * How comfortable are you interrupting the other person in either cases?
        3. If you are the leader,
            * How was that experience for you?
            * If the follower is lagging, how do you get an indication of that?
            * What do you do if you think the other person is lost?
            * How would you know if the other person is lost?
            * How have you communicated to the other person if you feel they were lost?
            * What have you done in the past to help the other person get unlost?

**Comment by magoolation** [comment]
Hi,
As a person with visual impairement I'm really interested in helping you on improvements  improvments to the product.
It's really hard to follow other colaborators as we have multiple cursors in the screen and NVDA doesn't know how to handle it. 
It might be usefull to have a special mode for really following only the colaborator cursor as it might NVDA on handling it.
I didn't tested the voice chat of LiveShare. My previous experience was using Zoom meeting or following a friend in his live coding session on Twitch but I can try it later and bring it back to you.
About communicating with the colaborators I don't have any restriction on telling the person she is going too fast, or I'm lost or ask what line or file is she changing at the momment.
LiveShare is a great addition to the Visual Studio toolbox and I'm really looking foward to use it to work and also for trainning other people with disabilites.

**Comment by sana-ajani** [comment]
@venkateshpotluri @daytonellwanger FYI

@magoolation thank you for your input! if you are interested in giving us more feedback on VS Code accessibility, you should join this gitter channel: https://gitter.im/Microsoft/vscode-a11y Our team is directly discussing issues and suggestions on the channel and we hold regular calls with our community there for product feedback.

**Comment by isidorn** [comment]
Closing this as I believe this was covered in the accessibility call
